docs/UserGuide.md

```diff
@@ -54,35 +55,92 @@ not be so obvious!
+[Section 3.2 - Reading this document](#reading-this-document), you can find sevaral useful tips on how to read this guide. The following section,
+[Section 4 - Features](#features), documents the four main features in **tCheck**, namely:
+
+   * Sales Tracking
+
+   * Ingredients Tracking
+
+   * Employee Management
+
+   * Other general features
+
+### 3.2 Reading this document <a name="reading-this-document"></a>
+
+This section introduces you to some key technical terms, symbols and syntax that are used throughout this guide.
+You may want to familarize yourself with them before moving on to the next section.
+
+#### 3.2.1 Terminology related to the GUI
+
+Figure 2 shows the GUI of **tCheck**, annotated with descriptions for all components of the GUI.
+
+<img src="images/tCheckInfographic.png" />
+Figure 2 - Annotated GUI of tCheck
+
+#### 3.2.4 Prefix process and usage
+
+The table below explains some important features of how prefix(es) are processed in tCheck to help you understand and use prefix(es) in commands.
+
+Situation | What will happen
+-------|------------------------------
+Duplicate (2 or more) valid prefixes detected | Only the last prefix and its following parameter are accepted and processed as part of the command.
+Invalid prefix entered | tCheck will not recognize this "prefix" and thus it will be processed as part of the parameter for the nearest previous valid prefix, which may result in error for the parameter.
+Unexpected prefix(es) entered for commands which do not take in any parameter | tCheck will not accept or process these prefix(es) because all words entered after the command words will be ignored. 
+
+**Example 1:**
+`i-set i/Milk i/Boba m/90`
+**How is the command processed:**
+In this case, the situation is duplicate valid prefixes detected. Hence, only the last, in this case, the second prefix and its following parameters are accepted.
+Thus, the command will be processed in the same way as `i-set i/Boba m/90`.
+
+**Example 2:**
+`i-set-all M/10 P/10 B/10 L/10 G/10 S/10 T/10`
+**How is the command processed:**
+In this example, the prefixes `M/`, `P/`, `B/`, `L`, `G` and `S/` are valid but `T/` is invalid. Hence, the situation is
+invalid prefix entered. Since tCheck will not be able to recognize the `T/` entered as a prefix. Thus, it will be treated as
+part of the parameter for the nearest previous prefix, which is `S/`. This will cause the command to fail because `10 T/20` is an invalid format for amount and appropriate error message will be shown.
+
+**Example 3:**
+`help i/Milk i don't know what to do `
+**How is the command processed:**
+In this case, `help` is a command which does not take in amy parameter. Hence, the situation is unexpected prefix(es) entered for commands which do not take in any parameter.
+Thus, tCheck will ignore all words come after the command word `help` and the command will be processed in the same way as `help`.
```

Comment:
The sentences that contains "the situation is ... " is not fluent. Perhaps a better way to structure the sentence is: "there is an .... (describe the situation)". e.g change could be: "there is a duplication of valid prefixes." Alternatively, eg. "the situation faced is the duplication of valid prefixes". I prefer the first suggestion.

Reply:
I don't think so. This is because I used the header "situation" in the table for explanation so I believe "the situation is" here will make more sense to users because it is extra clear that I am referring back to the table. Hence, I will not change the sentence to avoid "the situation is" . Thanks.

Reply:
Oh, okay I understand. I also think I figured out why it didn't sound fluent to me. I believe the sentence should be structured as this instead: "the situation of duplicate valid prefixes is detected". It was to do with the ending, rather than pertaining to "situation".

Reply:
Noted, I think changing the structure like this is okay, will change!